frontend/components/organisms/documents/DocumentExportForm.vue

```diff
@@ -37,6 +37,12 @@
         </v-sheet>
         <h2>{{ $t('dataset.exportDataMessage2') }}</h2>
         <v-text-field v-model="selectedFileName" placeholder="Name the file" />
+        <v-checkbox
+          v-model="onlyApproved"
+          label="Export only approved datasets"
```

Comment:
i would replace "datasets" with "documents" or "items". Ready for upstream otherwise.